www/ssc/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   cmake 1.1
+revision                    0
+
+github.setup                devongarde ssc 0.1.56 v
+github.tarball_from         releases
+categories                  www
+platforms                   {darwin >= 18} {openbsd >= 7.2}
+supported_archs             x86_64 arm64
+license                     GPL-3
+maintainers                 { @devongarde } openmaintainer
+description                 an opinionated static website nitpicker
+long_description            ${name}, the Static Site Checker, is an opinionated nitpicker for static \
+                            websites, analysing HTML, SVG, MathML, and more, including ontologies. \
+                            It can produce detailed reports, 'repaired' HTML with resolved server \
+                            side includes, site statistics, etc.. Consider it, at best, alpha.
+
+distname                    ${name}-v${version}
+distfiles                   ssc-v${version}.tgz
+extract.suffix              .tgz
+extract.post_args           "| tar -xzf -"
```

As Herby pointed out, you don't need two of those lines.

```suggestion
distname                    ${name}-v${version}
extract.suffix              .tgz
```